templates/classpath-jars-pv.yaml

```
@@ -48,3 +49,4 @@ spec:
     volumeAttributes:
       path: {{ include "palisade.deployment.path" . }}
   {{- end }}
+  {{- end }}
```

indent the first end to be inline with the else if
leave the 2nd end un-indented
```suggestion
  {{- end }}
{{- end }}
```

---

Thanks, got it.